_documentation/05-User-Manual.md

```diff
@@ -1,5 +1,32 @@
 ---
 title: User Manual
+toc: true
+toc_sticky: true
+h_range: [1,3]
 ---
 
-This document will be an user manual, containing information about setting up cups and printer application drivers.
\ No newline at end of file
+**This document will be an user manual, containing information about installing printer/scanner application drivers, installing CUPS-Snap, configuring and using Web Interface options and finally use the printer or scanner to print and scan respectively.**
+
+## Introduction
+
+The user is relieved from most of the complexities as compared to the manufacturer and OpenPrinting in this switch to new technology. They are just required to install Printer/Scanner Application Driver (based on the manufacturer and model) and use the same to get their devices working.
```

Comment:
The user is relieved from most of the complexities as compared to the manufacturer and OpenPrinting in this switch to new technology. They are just required to install **the driver provided as a Printer/Scanner Application** (based on the manufacturer and model) and use the same to get their devices working.